pythonosc/osc_packet.py

```diff
@@ -68,7 +68,14 @@ def __init__(self, dgram: bytes) -> None:
                     key=lambda x: x.time,
                 )
             elif osc_message.OscMessage.dgram_is_message(dgram):
-                self._messages = [TimedMessage(now, osc_message.OscMessage(dgram))]
+                self._messages = []
+                while(True):
+                    msg = osc_message.OscMessage(dgram)
+                    self._messages.append(TimedMessage(now, msg))
+                    if msg._dgram_tail is None:
```

Comment:
nit. you can just use `if not msg.dgram_tail:`,  no need to compare against None